### DISCUSSION TITLE
how i can play jak 3

### DISCUSSION THREAD

**Discussion by iliasdevpas** [open-goal/jak-project#3173] [Q&A] [answered]
hello i see you have started jak3 how i can play the development version?

**Comment by dallmeyer** [comment]
You can't yet.

**Reply by iliasdevpas** [reply]
when you add the maps i can play the jak3 developmet version?????

**Comment by xTVaser** [comment, marked as answer]
Please stop routinely trying to ban evade in the discord, and now also moving onto our github
![image](https://github.com/open-goal/jak-project/assets/13153231/c59e153d-62e8-4e8c-ba86-318fd7eab93f)

You have tried to join the discord with a half-dozen new accounts over the last month.
